x/feegrant/keeper/keeper.go

```diff
@@ -75,9 +75,9 @@ func (k Keeper) GrantFeeAllowance(ctx sdk.Context, granter, grantee sdk.AccAddre
 func (k Keeper) RevokeFeeAllowance(ctx sdk.Context, granter, grantee sdk.AccAddress) error {
```

Comment:
since `RevokeFeeAllowance` is only used internally, why not just having it at the `msgServer` level instead?

Reply:
It is also used in `UseGrantedFees`. Maybe we can make it private.